constructs/cluster.ts

```diff
@@ -113,8 +113,8 @@ export class Cluster extends Construct {
     const task = new EcsTaskDefinition(this, name, {
       family: name, // name used to group the definitions versions
 
-      cpu: taskDefinitionConfig.cpu ?? '1024',
-      memory: taskDefinitionConfig.memory ?? '3072',
+      cpu: taskDefinitionConfig.cpu ?? '256',
+      memory: taskDefinitionConfig.memory ?? '512',
```

Comment:
Since you changed the default, maybe you want to set values for `ecsConfig.graasp` at least for production. Not sure if it has to be very high or not.

Reply:
The values for the `graasp` service are overridden in the taskdef pushed when we deploy in the repo. So they have no impact.